### DISCUSSION TITLE
Divide presentation / reduce size

### DISCUSSION THREAD

**Discussion by sebj8** [senshu/Sozi#540] [Q&A]
I did the presentation with a lot of images, unfortunately the presentation grew to around 60MB. Is there any way to break the presentation down into smaller fragments? Is there any other way to reduce the size? Does the presentation have to be downloaded in full at once?

**Comment by aumouvantsillage** [comment]
> Is there any way to break the presentation down into smaller fragments?

The only technique available today is to create several independent presentations and use [hyperlinks](https://sozi.baierouge.fr/pages/tutorial-links.html#creating-hyperlinks-in-a-sozi-presentation) to chain them.

The drawback is that there will be no animation between the last frame of a presentation and the first frame of the next.

> Is there any other way to reduce the size?

The problem is the size of your SVG document. This is not directly related to Sozi.

If you haven't done it yet, you may consider resizing the images. For instance, if you know that your presentation will be played on a full-HD display, and if you don't need to zoom into your images, you could shrink them to a height of 1080 pixels or lower.

> Does the presentation have to be downloaded in full at once?

Currently yes.
When loading your presentation, the web browser cannot determine the appropriate moment to load each image, so it loads everything.
